app/src/main/java/org/ea/sqrl/activites/SimplifiedActivity.java

```diff
@@ -78,6 +82,15 @@ protected void onCreate(Bundle savedInstanceState) {
                 integrator.initiateScan();
             }
         );
+
+        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
This is actually not required here but as the feature should not be available for older devices we could check.

Reply:
You are correct, this is not required.  I think this must have been a vestige from when I had *application shortcut* code in that location.